pom.xml

```diff
@@ -28,21 +28,19 @@
   </licenses>
 
   <developers>
-    <developer>
-      <id>wenns</id>
-      <name>Waleri Enns</name>
-    </developer>
-    <developer>
-      <id>godin</id>
-      <name>Evgeny Mandrikov</name>
-      <organization>SonarSource</organization>
-    </developer>
-    <developer>
-      <id>lindamartin</id>
-      <name>Linda Martin</name>
-      <email>linda.martin@sonarsource.com</email>
-      <organization>SonarSource</organization>
-    </developer>
+    <!-- Inception & Historical Contributors -->
+    <developer><name>Andrea Guarino</name>      <organization>SonarSource</organization> <id>andreaguarino</id></developer>
+    <developer><name>Evgeny Mandrikov</name>    <organization>SonarSource</organization> <id>godin</id></developer>
+    <developer><name>Linda Martin</name>        <organization>SonarSource</organization> <id>lindamartin</id></developer>
+    <developer><name>Pierre-Yves Nicolas</name> <organization>SonarSource</organization> <id>pynicolas</id></developer>
+    <developer><name>Waleri Enns</name>                                                  <id>wenns</id></developer>
+    <!-- 2025 -->
```

Comment:
I have the feeling mentioning "2025" is a recipe to force ourselves to update this quite often (and somehow implies we didn't contribute before that?).
What about "active contributors" instead? Or "current contributors"?

Reply:
I'll use "active contributors" instead of "2025" indeed.